test/python/mock/test_configurable_backend.py

```diff
@@ -56,6 +56,8 @@ def test_set_parameters(self):
                 self.assertEqual(properties.backend_name, "Tashkent")
                 self.assertEqual(len(properties.qubits), n_qubits)
                 self.assertEqual(len(properties.gates), n_qubits)
+                self.assertAlmostEqual(properties.t1(0), 9.9e-5, places=7)
+                self.assertAlmostEqual(properties.t2(0), 14.6e-5, places=7)
```

Comment:
A very minor nitpick, but when we're doing scientific notation with SI prefixes, we'd usually use powers of 1,000 to do the scaling - `99e-6` seconds is clearly (once you're used to the convention) 99µs, but I have to think for a little bit what `9.9e-5` means.

Reply:
Ack